src/index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
```

Comment:
🤔  shouldn't it be scss?

Reply:
When i connect index.html with scss the whole page is blank. 
I fixed everything else, can i push commit without it?  ( Ir doesnt want to work) :/ )

Reply:
sure

Reply:
I uploaded bad version,  definitive i 'v3'